python/graphstorm/config/argument.py

```diff
@@ -1236,6 +1237,16 @@ def target_ntype(self):
             "Must provide the target ntype through target_ntype"
         return self._target_ntype
 
+    @property
+    def eval_target_ntype(self):
```

Comment:
is this a new parameter?

Reply:
this modification is better to be in another PR. the PR is to add an example. it should not modify the core code.

Reply:
I remove those changes from this PR. I will create a new PR later.